nimsuggest/nimsuggest.nim

```diff
@@ -198,17 +209,6 @@ proc execute(cmd: IdeCmd, file, dirtyfile: string, line, col: int;
     else:
       localError(gTrackPos, "found no symbol at this position " & $gTrackPos)
 
-proc executeEpc(cmd: IdeCmd, args: SexpNode;
```

Comment:
Why remove this?

Reply:
I'm sorry if I was missing something. Is the proc still using? I searched in the nimsuggest.nim, but 
no place is used. Also after I removed the proc, I could use nimsuggest's completion, linter, or other stuff correctly on Emacs, so I thought you forgot to remove this.

Reply:
Ok.